src/types/FormTypes.ts

```diff
@@ -38,13 +44,15 @@ export interface Banner {
 export interface Step {
   title: string;
   description: string;
-  id?: string;
+  id: string;
   group: string;
   questions?: Question[];
   actions?: Action[];
   banner?: Banner;
 }
 
+export type StepperActions = 'next' | 'back' | 'up' | 'down' | 'none';
```

Comment:
Since we now export this type from the helper, we should probably only use that type everywhere, so this should be removed.